docs/quickstart.md

```diff
@@ -54,9 +54,11 @@ Once you have Helm ready, you can initialize the local CLI and also
 install Tiller into your Kubernetes cluster in one step:
 
 ```console
-$ helm init
+$ helm init --max-history 200
```

Comment:
I wonder should `--max-history 200` be moved into the tip are like the other tips. You can add it as an example setting of 200.

Reply:
The whole point of this PR is to make people aware to run it on the init, because pruning the tiller history after you forget to run it during the first ever init is the pain in the ass.

Reply:
I understand and appreciate that. I think it is well worth adding. 

The point I am making is that it should follow the pattern of the other `Tips` and be added underneath as you have done and not added to the `$ helm init` command.

Reply:
I don't think most people will read the tips before running helm init for the first time. The whole point is to make them aware of `--max-history` before running helm init.